app/src/main/java/com/zeapo/pwdstore/UserPreference.kt

```diff
@@ -374,6 +383,23 @@ class UserPreference : AppCompatActivity() {
             }
         }
 
+        private fun updateClearSavedPassphrasePrefs() {
+            clearSavedPassPreference?.apply {
+                val sshPass = encryptedPreferences.getString("ssh_key_local_passphrase", null)
+                val httpsPass = encryptedPreferences.getString("https_password", null)
+                if (sshPass == null && httpsPass == null) {
+                    isVisible = false
+                    return
```

Comment:
Maybe this should be annotated with an `@apply`? Otherwise we are in for a nasty bug if we should ever add anything else to this function.

Reply:
That makes sense, can you make the change? I've fixed repository settings so you'll be able to also merge this when presubmit runs finish.

Reply:
Sure!